java/runtime/src/main/java/org/ray/runtime/task/TaskSpec.java

```diff
@@ -50,7 +51,10 @@
   // Task arguments.
   public final FunctionArg[] args;
 
-  // return ids
+  // number of return objects.
+  public final int numReturns;
```

Comment:
Do we really need this member `numReturns `? I think we can use `returnIds.length` instead of it.

Reply:
@jovany-wang We would better to keep this to align with the `gcs.fbs`? `returnIds` is not in `gcs.fbs` now but it is called in many places, so I kept it.